test/instance.ts

```diff
@@ -1273,7 +1294,7 @@ describe('Instance', () => {
     it('should accept gaxOptions', done => {
       const gaxOptions = {};
       instance.request = config => {
-        assert.strictEqual(config.gaxOpts, gaxOptions);
+        assert.deepStrictEqual(config.gaxOpts, gaxOptions);
```

Comment:
```suggestion
        assert.strictEqual(config.gaxOpts, gaxOptions);
```

If after making that change, the test fails, then we would want to keep `deepStrictEqual`, but also set a property on `gaxOptions`.